scss/_contact.scss

```diff
@@ -1,83 +1,83 @@
-.contact {
-  padding: 7.1% 6% 8.4% 6%;
-  background: url(./../images/contact_us/map-image.jpg) center/cover no-repeat;
-}
-
-.contact__heading {
-  color: #fff;
-  @include section-heading;
-}
-
-.contact__main-paragraph {
-  padding: 0.3% 0 5.8% 0;
-  @include paragraph-under-main-heading;
-}
-
-.contact__form {
-  max-width: 75rem;
-  width: 100%;
-  margin: 0 auto;
-  font-family: $font-main;
-}
-
-.contact__sender {
-  display: block;
-  width: 100%;
-  border: none;
-  font-family: inherit;
-  padding-bottom: 6.5%;
-  @include respond-to(medium) {
-    padding-bottom: 0;
-    display: inline-block;
-    width: 48.5%;
-  }
-}
-
-.contact__data {
-  display: block;
-  font-family: inherit;
-  border: 1px solid #ccc;
-  border-radius: 4px;
-  background-color: #fff;
-  width: 100%;
-  height: 3.225rem;
-  text-indent: 1.25rem;
-  &:nth-of-type(1) {
-    margin-bottom: 1.8rem;
-  }
-  &:nth-of-type(2) {
-    margin-bottom: 1.8rem;
-  }
-  &::placeholder {
-    @include input-placeholder;
-  }
-}
-
-.contact__message {
-  display: block;
-  width: 100%;
-  height: 13.23rem;
-  padding-top: 3%;
-  border: none;
-  border-radius: 4px;
-  font-family: inherit;
-  resize: none;
-  text-indent: 1.25rem;
-  &::placeholder {
-    @include input-placeholder;
-  }
-  @include respond-to(medium) {
-    width: 48.5%;
-    float: right;
-  }
-}
-
-.contact__btn {
-  display: block;
-  margin: 0 auto;
-  margin-top: 4%;
-  padding: 1.35rem 2.8rem;
-  cursor: pointer;
-  @include button-big;
-}
-
+.contact {
+  padding: 7.1% 6% 8.4% 6%;
+  background: url(./../images/contact_us/map-image.jpg) center/cover no-repeat;
+}
+
+.contact__heading {
+  color: #fff;
+  @include section-heading;
+}
+
+.contact__main-paragraph {
+  padding: 0.3% 0 5.8% 0;
+  @include paragraph-under-main-heading;
+}
+
+.contact__form {
+  max-width: 75rem;
+  width: 100%;
+  margin: 0 auto;
+  font-family: $font-main;
+}
+
+.contact__sender {
+  display: block;
+  width: 100%;
+  border: none;
+  font-family: inherit;
+  padding-bottom: 6.5%;
+  @include respond-to(medium) {
+    padding-bottom: 0;
+    display: inline-block;
+    width: 48.5%;
+  }
+}
+
+.contact__data {
+  display: block;
+  font-family: inherit;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  background-color: #fff;
+  width: 100%;
+  height: 3.225rem;
+  text-indent: 1.25rem;
+  &:nth-of-type(1) {

```

Comment:
You can use `&:nth-of-type(1n)` here.
Then you can remove whole `&:nth-of-type(2)` below